dashboards-observability/common/constants/colors.ts

```diff
@@ -166,7 +166,7 @@ export const COLOR_PALETTES = [
     type: 'gradient',
   },
 ];
-export const HEX_CONTRAST_COLOR = 0xFFFFFF;
+export const HEX_CONTRAST_COLOR = 0xffffff;
```

Comment:
i checked implementation of this variable in pie chart. it always return  same value i.e. FFFF in case of single color palatte. may be pie chart is handling passed on color values internally. @spattnaik